settings.gradle

```diff
@@ -0,0 +1,2 @@
+include ":example-android-project"
+includeBuild "svg-2-android-vector"
```

Comment:
When I was trying to test the build before, I found having the projects connected would cause an initialization error as the gradle extension wasn't available when initializing the `example-android-project`. How were you building and testing the project? I always use the pattern that the `.travils.yml` and `validate_plugin` files do with `./gradlew -p`.

Reply:
I think I loaded the whole thing into android studio, as well as added it as `includeBuild` in my consuming project to test with. I followed the pattern from here: https://ncorti.com/blog/gradle-plugins-and-composite-builds  which I have recently used when contributing to AboutLibraries as well: https://github.com/mikepenz/AboutLibraries/pull/552 https://github.com/mikepenz/AboutLibraries/pull/555

Reply:
Ah, so I had missed a little detail - these lines are what make it work right (should be able to remove mavenLocal then too):

```
includeBuild("svg-2-android-vector") {
    dependencySubstitution {
        substitute module('com.quittle:svg-2-android-vector') with project(':')
    }
}
```